src/containerapp/azext_containerapp/containerapp_auth_decorator.py

```diff
@@ -116,3 +120,40 @@ def get_argument_excluded_paths(self):
 class ContainerAppPreviewAuthDecorator(ContainerAppAuthDecorator):
     def construct_payload(self):
         super().construct_payload()
+        self.set_up_token_store()
+
+    def set_up_token_store(self):
+        if self.get_argument_token_store() is None:
+            return
+
+        if self.get_argument_token_store() is False:
+            safe_set(self.existing_auth, "login", "tokenStore", "enabled", value=False)
+            return
+
+        safe_set(self.existing_auth, "login", "tokenStore", "enabled", value=True)
+
+        if self.get_argument_sas_url_secret() is None and self.get_argument_sas_url_secret_name() is None:
+            raise ArgumentUsageError('Usage Error: only blob storage token store is supported. --sas-url-secret and --sas-url-secret-name should provide exactly one when token store is enabled')
+        if self.get_argument_sas_url_secret() is not None and self.get_argument_sas_url_secret_name() is not None:
+            raise ArgumentUsageError('Usage Error: --sas-url-secret and --sas-url-secret-name cannot both be set')
+        if self.get_argument_sas_url_secret() is not None and not self.get_argument_yes():
+            msg = 'Configuring --sas-url-secret will add a secret to the containerapp. Are you sure you want to continue?'
```

Comment:
is it required to add prompt?

Reply:
if yes parameter is not passed, there is prompt to let user confirm it.

Reply:
It's the same behavior as others. For example: az containerapp microsoft update xxx